.env.profiling

```
@@ -53,6 +53,8 @@ SIMPLE_SERVER_DATABASE_HOST=localhost
 SIMPLE_SERVER_ENV=development
 SIMPLE_SERVER_HOST='localhost:3000'
 SIMPLE_SERVER_HOST_PROTOCOL=https
+STALE_EXPERIMENT_VISITED_SINCE_DAYS=365
+STALE_EXPERIMENT_VISITED_UNTIL_DAYS=35
```

I am not sure how to interpret these envvar names.

The way I'm reading it right now is: "Stale patients visited since 365 days, until 35 days". Which does not make sense.

Riff on an alternatives:

```suggestion
STALE_EXPERIMENT_MIN_DAYS_SINCE_LAST_VISIT=35
STALE_EXPERIMENT_MAX_DAYS_SINCE_LAST_VISIT=365
```

It's quite possible that the current envvar name hasn't clicked for me for whatever reason and the name is fine. An explanation would help.

---

This name is much clearer. Updated